Alune.spec

```diff
@@ -0,0 +1,39 @@
+# -*- mode: python ; coding: utf-8 -*-
+
+
+a = Analysis(
+    ['main.py'],
+    pathex=[],
+    binaries=[],
+    datas=[('alune/images', 'alune/images')],
+    hiddenimports=[],
+    hookspath=[],
+    hooksconfig={},
+    runtime_hooks=[],
+    excludes=[],
+    noarchive=False,
+    optimize=0,
```

Comment:
Any reason not to further optimize the code and reduce the file size?
https://pyinstaller.org/en/stable/feature-notes.html?highlight=bytecode%20optimization#bytecode-optimization-level

Reply:
For now if that's all, let's just leave it yeah! 😛